packages/grid/src/vaadin-grid-styles.js

```diff
@@ -116,6 +116,7 @@ registerStyles(
       width: 100%;
       box-sizing: border-box;
       margin: 0;
+      pointer-events: none;
```

Comment:
A test case that clicks an element found at the empty space between the last frozen and first frozen-to-end (using `shadowRoot.elementFromPoint()`) would be nice.

Reply:
Or `await sendMouse({ type: 'click', position: [xCoordinate, yCoordinate] });` that you already used 👍